Jenkinsfile.integration

```diff
@@ -56,23 +58,24 @@ pipeline {
                     sh "git fetch --no-tags"
                     List<String> sourceChanged = sh(returnStdout: true, script: "git diff --name-only origin/master..origin/${env.BRANCH_NAME}").split()
                     echo "Files changed for this PR:\n${sourceChanged.join('\n')}"
-                    for (int i = 0; i < sourceChanged.size(); i++) {
-                        if (sourceChanged[i].contains("playbooks/")) {
-                            PlaybooksChanged = true
-                        }
-                        if (sourceChanged[i].contains("doc/")) {
-                            DocsChanged = true
-                        }
-                        if (sourceChanged[i].contains("tox.ini")) {
-                            ToxChanged = true
-                        }
-                    }
+                    PlaybooksChanged = sourceChanged.any{it.contains("doc/")}
+                    DocsChanged = sourceChanged.any{it.contains("playbooks/")}
```

Comment:
These appear backwards: DocsChanged should check the "doc/" directory, and PlaybooksChanged should check the "playbooks/" directory.

Reply:
done!